src/main/kotlin/no/nav/familie/ef/personhendelse/client/OppgaveClient.kt

```diff
@@ -55,7 +55,8 @@ class OppgaveClient(
     fun finnMapper(finnMappeRequest: FinnMappeRequest): FinnMappeResponseDto {
         val response = getForEntity<Ressurs<FinnMappeResponseDto>>(
             UriComponentsBuilder.fromUri(URI.create("$oppgaveUrl/mappe/sok"))
-                .queryParams(finnMappeRequest.toQueryParams())
+                .queryParam("enhetsnr", finnMappeRequest.enhetsnr)
+                .queryParam("limit", finnMappeRequest.limit)
```

Comment:
Vet ikke helt hva jeg synes om dette, jeg som konsument av "FinnMappeRequest" et annet sted i koden forventer at alle felter blir brukt.. ? 
Det som kan skje er att man setter noen av de andre feltene men då ikke blir sendt videre som query params

Reply:
Noterer meg at det kan refaktoreres til enten færre felter i request-objektet, eller å gjøre om metoden til å ta inn to parametere som brukes, og godkjenner denne :)